deployment/ocp/cloudigrade.yml

```diff
@@ -8,7 +8,123 @@ objects:
 - kind: DeploymentConfig
   apiVersion: v1
   metadata:
-    name: cloudigrade
+    name: ${NAME}-celery
+  spec:
+    replicas: 1
```

Comment:
Should this be configurable? Does this container support multiple pods running? Should we define [autoscaling](https://docs.openshift.com/dedicated/dev_guide/pod_autoscaling.html)?